cmd/zed/zed.d/zed.rc

```diff
@@ -86,6 +86,14 @@
 #ZED_SPARE_ON_IO_ERRORS=1
 
 ##
+# Turn on/off enclosure LEDs when drives get DEGRADED/FAULTED.  This works for
+# device mapper and multipath devices as well.  Your enclosure must be
+# be supported by the Linux SES driver for this to work.
```

Comment:
Well we might as well fix it now.  @tonyhutter can you make this quick fix, I also noticed several places where this patch introduce an extra line of whitespace which isn't needed.  It would be good to fix those at the same time.